src/chttpd/src/chttpd_misc.erl

```diff
@@ -103,6 +103,52 @@ maybe_add_csp_headers(Headers, _) ->
     Headers.
 
 handle_all_dbs_req(#httpd{method='GET'}=Req) ->
+    Params = lists:flatmap(fun({K, V}) -> parse_all_dbs_param(K, V) end,
+        chttpd:qs(Req)),
+    Args = lists:foldl(fun({K, V}, Arg) -> validate_alldbs_query(K, V, Arg)
```

Comment:
Same here. If you change validate_alldbs_query to accept tuple and accumulator you could write:

```
Args = lists:foldl(fun validate_alldbs_query/2, Params),
```